src/server/server.ts

```diff
@@ -210,6 +212,40 @@ registerRoute('post', '/compile/snippet', compileCommon);
  */
 registerRoute('post', '/compile/page', (req, res) => compileCommon(req, res, true /*wrapWithRunnerChrome*/));
 
+registerRoute('get', '/open-in-playground/:host/:type/:id/:filename', async (req, res) => {
```

Comment:
Also, can we add a server-based log entry, and also generate a GUID that we can stuff into the document like a correlation ID, to understand our funneling of users (from view to download to actually have open)